code/modules/movespeed/modifiers/status_effects.dm

```diff
@@ -5,7 +5,7 @@
 	multiplicative_slowdown = 3
 
 /datum/movespeed_modifier/status_effect/lightpink
-	multiplicative_slowdown = -0.5
+	multiplicative_slowdown = -0.1
```

Comment:
It's only there to offset the speed penalty from your armor now. At 0.1 it's likely not going to make your movement tick faster, but it will dampen the impact of any slowdown you have left. But I suppose 0.25 is fine with the nerfs to red potions.

Reply:
has This been addressed